3.workload.md

```diff
@@ -0,0 +1,99 @@
+# 3. Workload
+
+This section defines the workload definition schematics.
+
+Usually, workload definitions are provided by the OAM runtime (just as traits are) so that application developers may inspect the OAM runtime and learn what workloads are available for use.  
+
+## Top-Level Attributes
+
+The following attributes are common across all schemata defined in this document. They follow the [Kubernetes specification](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#required-fields).
```

Comment:
I think MSFT/Alibaba/Upbound should find time this week to discuss this. I think we should address this in a follow-up PR since this is only getting into alpha2 branch. This PR is more for the technical part.